simplified-app-shared/src/main/java/org/nypl/simplified/app/MainSettingsAccountActivity.java

```diff
@@ -579,17 +579,39 @@ public void onClick(final DialogInterface dialog, final int which) {
 
     if (this.account.supportsCardCreator() && locationpermission) {
 
-    in_signup.setOnClickListener(
-      new OnClickListener() {
-        @Override
-        public void onClick(
-          final @Nullable View v) {
-          final Intent cardcreator = new Intent(MainSettingsAccountActivity.this, CardCreatorActivity.class);
-          startActivity(cardcreator);
-        }
-      });
+      in_signup.setOnClickListener(
+        new OnClickListener() {
+          @Override
+          public void onClick(
+            final @Nullable View v) {
+            final Intent cardcreator = new Intent(MainSettingsAccountActivity.this, CardCreatorActivity.class);
+            startActivity(cardcreator);
+          }
+        });
+      in_signup.setText("Sign Up");
```

Comment:
line 591 should be deleted, 592 does the same thing and references the string resource file.